src/pykeen/triples/splitting.py

```diff
@@ -317,48 +485,14 @@ def split(
         ratios = [0.8, 0.1, 0.1]  # also makes a [0.8, 0.1, 0.1] split
         train, test, val = split(triples, ratios)
     """
-    if method is None:
-        method = "coverage"
-    if method not in SPLIT_METHODS:
-        raise ValueError(f'Invalid split method: "{method}". Allowed are {SPLIT_METHODS}')
-
-    random_state = ensure_torch_random_state(random_state)
-    ratios = normalize_ratios(ratios=ratios)
-    sizes = get_absolute_split_sizes(n_total=mapped_triples.shape[0], ratios=ratios)
-
-    if method == "cleanup":
-        triples_groups = _split_triples(
-            mapped_triples,
-            sizes=sizes,
-            random_state=random_state,
-        )
-        # Make sure that the first element has all the right stuff in it
-        logger.debug("cleaning up groups")
-        triples_groups = _tf_cleanup_all(triples_groups, random_state=random_state if randomize_cleanup else None)
-        logger.debug("done cleaning up groups")
-    elif method == "coverage" or method is None:
-        seed_mask = _get_cover_deterministic(triples=mapped_triples)
-        train_seed = mapped_triples[seed_mask]
-        remaining_triples = mapped_triples[~seed_mask]
-        if train_seed.shape[0] > sizes[0]:
-            raise ValueError(f"Could not find a coverage of all entities and relation with only {sizes[0]} triples.")
-        remaining_sizes = (sizes[0] - train_seed.shape[0],) + tuple(sizes[1:])
-        train, *rest = _split_triples(
-            mapped_triples=remaining_triples,
-            sizes=remaining_sizes,
-            random_state=random_state,
-        )
-        triples_groups = [torch.cat([train_seed, train], dim=0), *rest]
-    else:
-        raise ValueError(f"invalid method: {method}")
-
-    for i, (triples, exp_size, exp_ratio) in enumerate(zip(triples_groups, sizes, ratios)):
-        actual_size = triples.shape[0]
-        actual_ratio = actual_size / exp_size * exp_ratio
-        if actual_size != exp_size:
-            logger.warning(
-                f"Requested ratio[{i}]={exp_ratio:.3f} (equal to size {exp_size}), but got {actual_ratio:.3f} "
-                f"(equal to size {actual_size}) to ensure that all entities/relations occur in train.",
-            )
-
-    return triples_groups
+    # backwards compatibility
+    splitter_cls: Type[Splitter] = splitter_resolver.lookup(method)
+    kwargs = dict()
+    if splitter_cls is CleanupSplitter:
```

Comment:
maybe just 
```python
if splitter_cls is CleanupSplitter and randomize_cleanup:
    kwargs["cleaner" = cleaner_resolver.normalize_cls(RandomizedCleaner)
```


since the deterministic one is default anyway

Reply:
5cc5f90